broadcastevents/events.go

```diff
@@ -95,3 +95,8 @@ type AlertChannelEvent struct {
 	Compliance       string `json:"compliance score has decreased,omitempty"`
 	NewAdmin         string `json:"new cluster admin was added,omitempty"`
 }
+
+type ScanWithoutAccessKeyEvent struct {
+	EventBase   `json:",inline"`
+	ClusterName string `json:"clusterName"`
```

Comment:
can we verify that it's the cluster long name? and will we be able to attach the helm version or it's complicating the flow?